### ISSUE TITLE
Orca doesn't announce misspell words

### ISSUE THREAD

**Issue by sthibaul** [mate-desktop/pluma#339]
As initially reported on https://mail.gnome.org/archives/orca-list/2018-June/msg00281.html

#### Expected behaviour

Misspelled word should be announced by screen readers

#### Actual behaviour

Mispelled words are not announced

#### Steps to reproduce the behaviour

- run the orca screen reader
- open pluma
- enable spell check verification from the tools menu
- write a mispelled word and press escape

#### MATE general version

1.20

#### Package version

1.20

#### Linux Distribution

Debian sid

#### Additional information:

From Joanmarie: “The way this feature works is that Orca listens for a text attributes changed event. It seems like we're not getting this from Pluma when the misspelled indicator appears. If the Pluma developers fix this bug, Orca should JustWork(tm).”

If moving cursor back to the misspelled word, Orca telles it is misspelled, so apparently some support already exists.
